src/documents/tutorial-gettingStartedWithInfusion/BasicComponentCreation-Components.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Basic Component Creation - Components
+layout: default
+category: Tutorials
+---
+
+---
+Part of the [Getting Started with Infusion Tutorial](GettingStartedWithInfusion.md)
+
+---
+
+Regardless of which grade of component you use, the basic structure will be the same. We'll use the simplest grade, a plain **component**, to illustrate what this structure is. In future pages explaining other grades, you'll see the same principles.
+
+The definition of a component involves two things:
+
+1. declare the component **grade** and any default values for the component's **options**. Options are used by integrators to customize the behaviour of a component.
+2. define any public **functions** that the component requires to do its work.
+
+## Grade and Default Options ##
+
+A component's grade and any default options are registered with the framework using a call to [`fluid.defaults`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L1519-L1539), which has two parameters: the component name and an object containing the defaults. The parent grades for the component are specified in an array in the defaults called `gradeNames`. For a **little component**, specify the grade as `fluid.component`:
```

Comment:
This paragraph refers to "little component" – we're not using that phrase anymore, are we?